core/service/src/engine/threshold/threshold_kms.rs

```diff
@@ -30,6 +30,9 @@ pub struct ThresholdKms<
     pub(crate) insecure_crs_generator: ICG,
     pub(crate) context_manager: CM,
     pub(crate) backup_operator: BO,
+    pub(crate) my_party_id: usize,
```

Comment:
do we need to pass this here? I think in the other endpoints we infer this info from the session

Reply:
tackled in https://github.com/zama-ai/kms/pull/89/commits/31cb4a99db4671589eaf4358fab986edb032dad3